collect_app/src/main/res/drawable/ic_crosshairs.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="48dp"
+        android:height="48dp"
+        android:viewportWidth="48"
+        android:viewportHeight="48">
+    <path
+        android:fillColor="#FF0099FF"
```

Comment:
also colors.xml

Is this color defined anywhere externally? Who came up with it? Might be nice to document.

Reply:
I picked it to approximately match the blue that is used for the location dot and the shaded accuracy circle in Google Maps.  So, in some sense it's a guess.  It looks pretty close to my eye though.

Reply:
Okay, I moved the colours into colors.xml and gave them all names.  Seemed like it wouldn't hurt to be thorough so I did them all.